assets/src/test/java/bisq/asset/coins/DSTRATest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.AbstractAssetTest;
+
+import org.junit.Test;
+
+public class DSTRATest extends AbstractAssetTest {
+
+    public DSTRATest() {
+        super(new DSTRA());
+    }
+
+    @Test
+    public void testValidAddresses() {
+        assertValidAddress("DGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd7");
+        assertValidAddress("DQcAKx5bFoeRwAEHE4EHQykyq8u2M1pwFa");
+    }
+
+    @Test
+    public void testInvalidAddresses() {
+        assertInvalidAddress("DGiwGS8n3tJZuKxUdWF6MyTYvv6xgDcyd77");
```

Comment:
Please format the code according to Bisq code style.

Reply:
Hi @blabno and thank you for checking my PR! 
After review I simply can't find the spot where I violated Bisq code style. I took a look at several examples (PIVXTest, ZcashTest, etc.) and it seems to me like my file is exactly looking the same.
Maybe you can guide me to what's wrong with my Test class. Only style reference for bisq I found is https://github.com/bisq-network/style and it doesn't contain anything regarding code style as far as I could see as it's pretty new.

Reply:
You have additional indentation here.

Reply:
Tried to fix it with IDEA (did the code with Notepad++ before) and I hope it's done now @blabno .
I can't see other alignment issues.
Squashed again, hoping for an ACK this time